### DISCUSSION TITLE
[Question] rand isn't used in ccmd/vela/main.go

### DISCUSSION THREAD

**Discussion by horis233** [kubevela/kubevela#931] [Q&A]
When I checked the code of the vela CLI, I didn't find where `rand.Seed(time.Now().UnixNano())` works in the code.
Is it a redundant code?

Please correct me if I am wrong. Thanks in advance.
 
https://github.com/oam-dev/kubevela/blob/master/cmd/vela/main.go#L12

**Comment by wonderflow** [comment]
Yes, It's redundant code. Thanks for reporting!

**Comment by tossmilestone** [comment]
`rand.Seed(time.Now().UnixNano())` is useful, it's a global random seed init method, will affect the usage of functions unders `rand` library.

**Comment by wonderflow** [comment]
Moving to discussion.
